shared/fs/common/pie-slice.js

```diff
@@ -0,0 +1,84 @@
+// @flow
+import * as Kb from '../../common-adapters'
+import * as React from 'react'
+import * as Styles from '../../styles/index'
+
+type Props = {|
+  degrees: number,
+  animated?: boolean,
```

Comment:
It seems this is only not true in stories, where you're using a separate `Kb.Animated` component. If you want to test animation in stories, I think having a stateful component in the storybook is a better way than using `Kb.Animated` directly in story, like [this](https://github.com/keybase/client/blob/91136b97814eacb8153c9e7b2d29b0cb92c92c01/shared/common-adapters/switch.stories.js#L32). WDYT?

Reply:
I left this as an option in case the animation is too slow or too CPU-intensive to use in the sync status icons, so we can remove it easily.

Reply:
Ah I see. Makes sense. Would you mind still changing the story to not use a separate `Kb.Animated`? So that it uses exactly the same behavior as in the app.

Reply:
Okay, I've changed the story so that instead of having a separate animated one, there is a button on each one to animate it forward by 72 degrees.